servers/zms/schema/zms_server.sql

```diff
@@ -90,6 +90,7 @@ CREATE TABLE IF NOT EXISTS `zms_server`.`role` (
   `group_expiry_days` INT NOT NULL DEFAULT 0,
   `description` VARCHAR(4096) NOT NULL DEFAULT '',
   `delete_protection` TINYINT(1) NOT NULL DEFAULT 0,
+  `max_members` INT NOT NULL DEFAULT 0,
```

Comment:
Should we have some other numbers as the default value instead of 0?

Reply:
No, we can't introduce backward incompatible changes. Value of 0 indicates there is no limit which is the current behavior.